### ISSUE TITLE
Android 6 - java.time.Instant crash

### ISSUE THREAD

**Issue by but4er007** [mindbox-cloud/android-sdk#376]
Android 6. Mindbox android sdk 2.6.2.
Multiple crash in logs:
```
Mindbox monitoring caught unhandled error
java.lang.NoClassDefFoundError: Failed resolution of: Ljava/time/Instant;
at cloud.mindbox.mobile_sdk.logger.MindboxLoggerImpl$saveLog$1.invokeSuspend(MindboxLoggerImpl.kt:107)
```
Class Instant added in api 26, your minimum sdk version - 19.
https://developer.android.com/reference/java/time/Instant.html

**Comment by enotniy** [comment]
@but4er007 Thank you for bugreport.
Moved the issue to https://github.com/mindbox-cloud/issues-web-mobile/issues/2685

**Comment by enotniy** [comment]
@but4er007 Fixed in [Release 2.6.3](https://github.com/mindbox-cloud/android-sdk/releases/tag/2.6.3)
